pkg/ebpf/c/oom-kill-kern.c

```diff
@@ -0,0 +1,44 @@
+#define KBUILD_MODNAME "foo"
```

Comment:
should this be here? 😂

Reply:
not sure why, but without this defined it fails..

Reply:
To be tested but I have the feeling that we will be able to get rid of that once we’ll have upgraded to a version of BCC that has iovisor/bcc#2974.
But that PR has been merged only 9 days ago. We don’t have it yet. 😉